content/docs/dvclive/api-reference/live/index.md

```diff
@@ -50,10 +46,16 @@ Its parameters are focused on configuring the behavior of subsequent calls to
 - `resume` - If `True`, DVCLive will try to read the previous `step` from the
   `path` directory and start from that point. _Default_: `False`.
 
+  ⚠️ Don't change the default value unless you are going to update the `step`
+  value during training.
```

Comment:
Maybe it can be explained more. For example: "If you are not using `steps`, don't use `True` since DVCLive will preserve the previous run's files to try to read the step value."

Reply:
Added but with some rephrasing please TAL